Assets/XML/Text/Alt_Timelines/Clockpunk/Clockpunk_CIV4GameText.xml

```diff
@@ -152,18 +152,17 @@
 	</TEXT>
 	<TEXT>
 		<Tag>TXT_KEY_TECH_CLOCKPUNK_PEDIA</Tag>
-		<English>[BOLD]Clockpunk[\BOLD], is a subgenre of speculative fiction which is similar to steampunk, but deviates in its technology. As with steampunk, it portrays advanced technology based on pre-modern designs, but rather than the steam power of the Industrial Age, the technology used is based on springs, clockwork and similar. Clockpunk is based very intensively on the works of Leonardo da Vinci and as such, it is typically set during the Renaissance. The idea is fairly new and very few works have been created in this genre. Clockpunk works include Pasquale's Angel by Paul J. McAuley, and Jack Faust by Michael Swanwick, as well as the upcoming novel Mainspring by Jay Lake. There are also clockpunk elements in the Marvel 1602 comic series and its sequels, and in the Discworld novels of Terry Pratchett, most notably a da Vinci space shuttle in The Last Hero. Benot Sokal's Syberia adventure game could also be considered clockpunk, as it features spring-based machines and clockwork automata.</English>
+		<English>[BOLD]Clockpunk[\BOLD] is a subgenre of speculative fiction which is similar to steampunk, but deviates in its technology. As with steampunk, it portrays advanced technology based on pre-modern designs, but rather than the steam power of the Industrial Age, the technology used is based on springs, clockwork and similar. Clockpunk is based very intensively on the works of Leonardo da Vinci and as such, it is typically set during the Renaissance. The idea is fairly new and very few works have been created in this genre. Clockpunk works include Pasquale's Angel by Paul J. McAuley, and Jack Faust by Michael Swanwick, as well as the upcoming novel Mainspring by Jay Lake. There are also clockpunk elements in the Marvel 1602 comic series and its sequels, and in the Discworld novels of Terry Pratchett, most notably a da Vinci space shuttle in The Last Hero. Benot Sokal's Syberia adventure game could also be considered clockpunk, as it features spring-based machines and clockwork automata.</English>
 	</TEXT>
 	<TEXT>
 		<Tag>TXT_KEY_TECH_CLOCKPUNK_QUOTE</Tag>
 		<English>"A clockwork heart can't replace the real thing." - Dru Pagliassotti, Clockwork Heart</English>
 		<French>"Un coeur d'horlogerie ne peut pas remplacer la chose réelle." - Dru Pagliassotti, Le Coeur d'Horlogerie</French>
 		<Polish>"Mechaniczne serce może zastąpić rzeczywiste." - Dru Pagliassotti, Clockwork Heart</Polish>
-	</TEXT>
+	</TEXT><!--
```

Comment:
Commenting text tags will only cause txt key issue, wchich is more spammyu

Reply:
Ok, I'll take this as an opportunity to rework the file, then. :) The only problem is that it'll take me a while.

Reply:
> 
> 
> Ok, I'll take this as an opportunity to rework the file, then. :) The only problem is that it'll take me a while.

Its not only this file - search for all >TBD< entries - there were thousand of them or so.